.github/workflows/validate-coding-standards.yml

```diff
@@ -28,6 +28,15 @@ jobs:
         with:
           python-version: "3.9"
 
+      - name: Install CodeQL
```

Comment:
Why not use the .github/actions/install-codeql action?

Reply:
We only need the CLI and not the standard library.
I also wanted to update the `install-codeql` action to use `gh codeql`, however we still rely on the repository being available as sibling to the CLI. Once we move over to packs, we can update the action.